osquery/sql/sqlite_string.cpp

```diff
@@ -152,6 +152,41 @@ static void ip4StringToDecimalFunc(sqlite3_context* context,
   }
 }
 
+/**
+ * @brief Convert a hexadecimal string to integer
+ */
+static void hexStringToDecimalFunc(sqlite3_context* context,
+                                   int argc,
+                                   sqlite3_value** argv) {
+  assert(argc == 1);
```

Comment:
Did you leave this intentionally?

Reply:
Copied style from functions above.

Reply:
Do you want me to remove this and/or the other asserts in this file?